fastlane/lib/fastlane/plugins/plugin_info_collector.rb

```diff
@@ -0,0 +1,67 @@
+module Fastlane
+  class PluginInfoCollector
+    def initialize(ui = PluginGeneratorUI.new)
+      @ui = ui
+    end
+
+    def collect_info
+      plugin_name = collect_plugin_name
+      author = collect_author
+
+      PluginInfo.new(plugin_name, author)
+    end
+
+    #
+    # Plugin name
+    #
+
+    def collect_plugin_name
+      plugin_name = nil
+      loop do
+        plugin_name = @ui.input("What would you like to be the name of your plugin?")
```

Comment:
At this point it's not clear if the plugin should be pre-fixed, right? Let's assume we build a plugin called `hockey_dsym_upload`:

The user could just enter `hockey_dsym_upload` and get the `fastlane-plugin-` prefix for free, right? If the user pre-fixes the input here, we should automatically detect that